src/ui.form-input-money/useFormatCurrency.ts

```diff
@@ -61,17 +69,19 @@ export default function useFormatCurrency(elementId, options) {
       const offset = currentValueOffsetLength - prevValueOffsetLength;
 
       formattedValue.value = localFormattedValue || toValue(options).prefix;
-      rawValue.value = FormatService.getRawValue(localFormattedValue, toValue(options));
+      rawValue.value = getRawValue(localFormattedValue, toValue(options));
 
       await nextTick(() => {
         if (localFormattedValue.length === cursorEnd) return;
```

Comment:
if (localFormattedValue.length === cursorEnd || !cursorStart || !cursorEnd) return;

and checks below may be removed